stock_valuation_fifo_lot/models/product.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2023 Ecosoft Co., Ltd (https://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import models
+from odoo.tools import float_is_zero
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    def _sort_by_all_candidates(self, all_candidates, sort_by):
+        """Hook function for other sort by"""
+        return all_candidates
+
+    def _get_all_candidates(self, company, sort_by=None):
```

Comment:
In odoo 16, we have this kind of function https://github.com/odoo/odoo/blob/4d154a9f08cb2204df8566ed5c1e4f5e59e5d212/addons/stock_account/models/product.py#L331.
Should we extend and passing sort_by as a context?